source/wp-content/plugins/wporg-developer-blog-cpts/wporg-developer-blog-cpts.php

```diff
@@ -0,0 +1,171 @@
+<?php
+/**
+ * Plugin Name: Developer Blog custom post types and taxonomies.
```

Comment:
This can be slightly shorthanded, since anyone seeing this should know "CPT", and the description has the details.

```suggestion
 * Plugin Name: Developer Blog CPTs
```